lib/index.js

```diff
@@ -47,78 +47,67 @@ module.exports = function loader (css, map) {
 
   validateOptions(require('./options.json'), options, 'PostCSS Loader')
 
-  const rc = {
-    path: path.dirname(file),
-    ctx: {
-      file: {
-        extname: path.extname(file),
-        dirname: path.dirname(file),
-        basename: path.basename(file)
-      },
-      options: {}
+  Promise.resolve().then(() => {
+    if (!options.config || (options.config && !options.config.path)) {
```

Comment:
@michael-ciniawsky two solution:
1. Load `postcss` config always and them merge option from loader with config options (i think loader options more important than `postcss` config option, but ).
2. Load `postcss` config only when `config: true` or `config: { path: '/path/to/config'}` (best way, no problems with perf and understandable using)